src/tests/unit/tests/electron/flux/action-creator/window-state-action-creator.test.ts

```diff
@@ -28,4 +29,17 @@ describe(WindowStateActionCreator, () => {
 
         setRouteActionMock.verifyAll();
     });
+
+    it('calling setRoute invokes setWindowState action with given payload', () => {
```

Comment:
test name seems off here (it mention calling `setRoute`)

Reply:
Copy paste strikes again